.github/actions/playwright-cache/action.yml

```
@@ -0,0 +1,44 @@
+---
+name: "Playwright Cache Action"
+description: "Initialize Playwright Cache"
+inputs:
+  version:
+    description: "Playwright version"
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: 🆙 Set env for os
+      shell: bash
+      env:
+        OS: ${{ runner.os }}
+      run: |
+        if [[ $OS == "Windows" ]]; then
+          echo "CACHE_PATH=C:\Users\runneradmin\AppData\Local\ms-playwright" >> "$GITHUB_ENV"
+          echo "BROWSERS=chromium firefox" >> "$GITHUB_ENV"
+          echo "OS=windows" >> "$GITHUB_ENV"
+        else
+          echo "CACHE_PATH=~/Library/Caches/ms-playwright" >> "$GITHUB_ENV"
+          echo "BROWSERS=webkit chromium firefox" >> "$GITHUB_ENV"
+          echo "OS=macos" >> "$GITHUB_ENV"
+        fi
+
+    - name: 🆒 Cache Playwright binaries
+      uses: actions/cache@v4
+      id: playwright-cache
+      with:
+        path: ${{ env.CACHE_PATH }}
+        key: "${{ runner.os }}-playwright-${{ inputs.version }}"
+        restore-keys: |
+          ${{ runner.os }}-playwright-
+
+    - name: 🎄🎸🥊 Log Cache Hit
+      shell: bash
+      env:
+        HIT: ${{ steps.playwright-cache.outputs.cache-hit }}
+      run: echo $HIT
+
+    - name: ⏬ Install Playwright's dependencies
+      shell: bash
+      if: steps.playwright-cache.outputs.cache-hit != 'true'
+      run: npx playwright install --with-deps ${{ env.BROWSERS }}
```

```suggestion
      run: npx --no -- playwright install --with-deps ${{ env.BROWSERS }}
```

We should use the playwright already installed and ensure that we're sticking to that for control over the playwrights version being in use.